src/arc_spice/__init__.py

```diff
@@ -1,5 +1,7 @@
 """
-ARC-SPICE: Sample Level, Pipeline Introduced Cumulative Errors (SPICE). Investigating methods for generalisable measurement of cumulative errors in multi-model (and multi-modal) ML pipelines.
+ARC-SPICE: Sample Level, Pipeline Introduced Cumulative Errors (SPICE).
+Investigating methods for generalisable measurement of cumulative errors in
+multi-model (and multi-modal) ML pipelines.
 """
 
 from __future__ import annotations
```

Comment:
not sure what this is for?

Reply:
ah sorry I see you didn't add this line in this PR but I was wondering what L7 was for and think it can be removed. If agree and convenient then please do before merge but happy to approve regardless.